### ISSUE TITLE
"invalid response data" in total lines

### ISSUE THREAD

**Issue by dishb** [badges/shields#9389]
### Are you experiencing an issue with...

shields.io

### 🐞 Description

I'm using the [`total lines`](https://shields.io/badges/lines-of-code) badge, and I'm getting an "invalid response data" error. Here is a screenshot:
![Screen Shot 2023-07-15 at 7 20 57 PM](https://github.com/badges/shields/assets/130259005/0a67336c-c3e6-4af0-9cb6-0d59e58274ff)

I've gotten this error before, so it's not like I had a bad day today 😅

As for the repository I'm trying to access, it can be viewed at [dishb/hive](https://github.com/dishb/hive). So the repository does indeed exist.

### 🔗 Link to the badge

Here is the badge in markdown:
![Lines of code](https://img.shields.io/tokei/lines/github/dishb/hive)

The code:
`![Lines of code](https://img.shields.io/tokei/lines/github/dishb/hive)`

The URL:
https://img.shields.io/tokei/lines/github/dishb/hive

**Comment by chris48s** [comment]
The reason this is failing is because the response from tokei does not have a `lines` property. The detailed error is `ValidationError: "lines" is required`

I am also seeing this on the responses for other repos. I'm going to raise an issue for this one upstream.

**Comment by dishb** [comment]
Okay, thank you.

**Comment by chris48s** [comment]
follow https://github.com/XAMPPRocky/tokei/issues/1017 for updates

**Comment by dishb** [comment]
Just checked the [API docs](https://docs.rs/tokei/latest/tokei/struct.CodeStats.html#structfield.code), maybe the correct way to access the lines of code from the response is through `code`?

**Comment by ashuthe1** [comment]
Hi! I am new to this community and would love to work on this.

**Comment by dishb** [comment]
@ashuthe1 - You'd probably need to ask to be assigned to [this issue.](https://github.com/XAMPPRocky/tokei_rs/issues/233). If you noticed, the `needs-upstream-help` means this isn't necessarily an issue with shields but with one of its service providers.

**Comment by calebcartwright** [comment]
Yeah appreciate the offer @ashuthe1 but there's nothing we can do here and are blocked on the upstream issue. 

If you're just interested in getting involved with Shields more generally there's a number of other issues you could potentially look at, and I'd suggest starting with these: https://github.com/badges/shields/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Comment by ashuthe1** [comment]
@calebcartwright thank you so much, I will look into good first issue first.

**Comment by dishb** [comment]
Duplicate of #7569.
